lms/djangoapps/mobile_api/models.py

```diff
@@ -1,10 +1,10 @@
 """
 ConfigurationModel for the mobile_api djangoapp.
 """
-
-from django.db.models.fields import TextField
-
+from django.db import models
+from lms.djangoapps.mobile_api import utils
```

Comment:
We should import modules from the same path specified in `INSTALLED_APPS`. Otherwise, in some cases Django app loading logic does not work properly. So this should just be `from mobile_api import utils` like below.